cloud/issue.go

```diff
@@ -981,9 +981,49 @@ func (s *IssueService) AddComment(ctx context.Context, issueID string, comment *
 	return responseComment, resp, nil
 }
 
+// GetComment returns a comment for issueID and commentID.
+//
+// Jira API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-comments/#api-rest-api-2-issue-issueidorkey-comment-id-get
+func (s *IssueService) GetComment(ctx context.Context, issueID, commentID string) (*Comment, *Response, error) {
+	apiEndpoint := fmt.Sprintf("rest/api/2/issue/%s/comment/%s", issueID, commentID)
+	req, err := s.client.NewRequest(ctx, http.MethodGet, apiEndpoint, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	responseComment := new(Comment)
+	resp, err := s.client.Do(req, &responseComment)
+	if err != nil {
+		jerr := NewJiraError(resp, err)
+		return nil, resp, jerr
+	}
+
+	return responseComment, resp, nil
+}
+
+// GetComments retrieves all comments for a given issueID.
+//
+// Jira API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-comments/#api-rest-api-2-issue-issueidorkey-comment-get
+func (s *IssueService) GetComments(ctx context.Context, issueID string) (*Comments, *Response, error) {
```

Comment:
Should we also add the optional API parameters like `startAt`, `maxResult`, etc.?
I think it would be great as introducing them later would be a breaking change.